examples/test_minimal/main.tf

```diff
@@ -0,0 +1,11 @@
+module "minimal" {
+  # source           = "git::https://github.com/Datatamer/terraform-aws-s3?ref=1.0.0"
+  source           = "../../"
+  bucket_name      = var.test_bucket_name
+  read_only_paths  = ["path/to/ro-folder"]
+  read_write_paths = ["path/to/rw-folder", "path/to/another-rw-folder"]
+}
+
+provider "aws" {
+  region = var.aws_region
+}
```

Comment:
One of our lint checks is to have a single trailing newline at the end of a file.

I'll add you to that repo so you should be able to run most of the `make lint`. I think the tf-sec is internally hosted so that may not run for you.